common/djangoapps/student/models.py

```diff
@@ -1863,7 +1865,7 @@ def _cert_name(self, course_name, cert_mode):
             cert_mode,
             _(u"{platform_name} Certificate for {course_name}")
         ).format(
-            platform_name=settings.PLATFORM_NAME,
+            platform_name=microsite.get_value('platform_name', settings.PLATFORM_NAME),
```

Comment:
No. I was not able to find platform name accessor. We can not use static here because this is going to be used in LinkedIn share url parameters.